code/modules/mob/living/carbon/xenomorph/castes/crusher/abilities_crusher.dm

```diff
@@ -97,11 +97,11 @@
 	X.face_atom(L) //Face towards the target so we don't look silly
 
 	var/facing = get_dir(X, L)
-	var/toss_distance = rand(3,5)
+	var/toss_distance = X.xeno_caste.crest_toss_distance //rand(3,5)//no more random.
 	var/turf/T = X.loc
 	var/turf/temp = X.loc
 	if(X.a_intent == INTENT_HARM) //If we use the ability on hurt intent, we throw them in front; otherwise we throw them behind.
-		for (var/x in 1 to toss_distance)
```

Comment:
Both types are acceptable so dont change this in future

Reply:
okie